src/recensio/plone/content/review_article_collection.py

```diff
@@ -48,8 +49,8 @@ class ReviewArticleCollection(Item):
     """Content-type class for IReviewArticleCollection."""
 
     def formatted_authors(self):
-        # TODO
-        return ""
+        authors_str = IAuthors(self).get_formatted_authors()
+        return authors_str
```

Comment:
It looks weird to me that we have a methof on a behavior and a facade to it in the content type class.
Can we have it only in one place?